cmd/buildah/run.go

```diff
@@ -137,10 +139,15 @@ func runCmd(c *cobra.Command, args []string, iopts runInputOptions) error {
 		}
 	}
 
+	if len(iopts.hostname) > 0 && iopts.noHostname {
+		return errors.New("--no-hostname and --hostname conflict, can not be used together")
```

Comment:
One's a setting in the UTS namespace, the other's a file.  _Do_ they conflict?

Reply:
I believe --hostname attempts to set the hostname field in /etc/hostname, if you don't think this would confuse the user, then I am fine with removing it.

Reply:
I'd remove the test